skyportal/handlers/api/candidate.py

```diff
@@ -1124,7 +1189,22 @@ def grab_query_results(
 
     items = []
     query_options = [joinedload(Obj.thumbnails)] if include_thumbnails else []
-    for item_id in page_ids:
-        items.append(Obj.query.options(query_options).get(item_id))
+
+    if len(page_ids) > 0:
```

Comment:
While we're here, this has always been a bit confusing to me -- can we rename `page_ids` to something more descriptive like `obj_ids_in_page` or something? I frequently find myself wondering what page IDs are :joy: